src/datarobotx/idp/__init__.py

```diff
@@ -5,3 +5,5 @@
 
 Ready for use in your orchestration layer of choice.
 """
+
+DEFAULT_MAX_WAIT = 30 * 60  # 30 minutes
```

Comment:
can the default come from the official client instead?

Reply:
we could import dr.enums.DEFAULT_MAX_WAIT - but it's only 600. I noticed some jobs frequently taking more than 20 minutes so opted to increase